groups/models.py

```diff
@@ -49,6 +49,18 @@ def unsubscribe(self, user):
     def is_subscribed(self, user):
         return self.watchers.filter(id=user.pk).exists()
 
+    def get_all_comments(self):
+        return self.discussions.comments()
+
+    def get_latest_comment(self):
+        return self.get_all_comments().latest('date_created')
+
+    def get_total_commenters(self):
+        return self.get_all_comments().users().count()
```

Comment:
Weirdly, I already had a test that claimed it already had protection from duplicates. I don't know if that test is broken, or if `count()` comes with a `distinct()` built-in, or what.